src/CharClass.ts

```diff
@@ -0,0 +1,34 @@
+import {Skill} from './Skill'
+import {Stat} from './Stat'
+import {Discipline} from './Discipline'
+import {Power} from './Power'
+
+const classes = ['Assassin', 'Barbarian', 'Crusader'] // Temporary list just to test setClass method.
```

Comment:
Try not to define test data in the real classes, you can define test data inside the test code.